app/diagnostics_test.go

```diff
@@ -98,6 +99,21 @@ func TestDiagnostics(t *testing.T) {
 		}
 	})
 
+	// create bot and bot post for testing bots in AnalyticsPostCountsByDay() and AnalyticsPostCount()
```

Comment:
Confused by this block -- it isn't clear to me where this particular data is /used/. Can you help me understand?

Reply:
@lieut-data, ahh..  sorry, this was something I added while test and totally forgot to delete.  Will remove.